templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -10,6 +10,17 @@ contents:
     # key files under /etc/NetworkManager/system-connections/
     # Managing key files is outside of the scope of this script
 
+    # if the interface is of type vmxnet3 add multicast capability for that driver
+    # BZ: 1854355
+    function configure_driver_options {
```

Comment:
I think you also need to call this at line 47 when br-ex is already configured. This has to be done on every boot.

Reply:
Ack.